docs/cluster-management.md

```diff
@@ -1 +1,43 @@
-// TODO
\ No newline at end of file
+# Multi-Cluster Management
+
+Multi-cluster management is a common task when it comes to managing your workloads on Amazon EKS. The management of these clusters adds a level of complexity especially when it comes to managing these clusters across multiple regions. Infrastructure as code (IaC) tools like the AWS CDK provides a seemless experience when it comes to the management of your clusters. 
+
+If you take a look at the main.ts file, you will notice that the stacks that define our Amazon EKS clusters are deployed to different regions as shown in the snippet below:
+
+```typescript
+new PipelineStack(app, "factory-pipeline", {
```

Comment:
where is this coming from?

Reply:
I just took a snippet from the main.ts file to show an example. I can totally expand on this as needed